mapping/sustainability.yaml

```
@@ -1135,10 +1132,7 @@
     module: Climate finance
     indicator: KPIs
     disclosure format: Disclose Key Performance Indicators adopted by the project (E.g. \[free text\]).
-    mapping: |-
-        Project level:
-
-        For each KPI add a `Metric` object to the `metrics` array and set the object's fields according to the schema. Prefix the metric's `.title` with "KPI".
+    mapping: 'Project level: For each KPI, add a [`Metric`](../../reference/schema.md#metric) object to the [`.metrics`](project-schema.json,,metrics) array and set its fields according to the schema. Prefix the metric''s [`.title`](project-schema.json,/definitions/Metric,title) with "KPI: ".'
```

```suggestion
    mapping: |
        Project level: 
        
        For each KPI, add a [`Metric`](../../reference/schema.md#metric) object to the [`metrics`](project-schema.json,,metrics) array and set its fields according to the schema. Prefix the metric's [`.title`](project-schema.json,/definitions/Metric,title) with "KPI: ".
```

moved this back just so that everything has the same look